### DISCUSSION TITLE
Issues related to sidebarCollapsible

### DISCUSSION THREAD

**Discussion by khushal87** [facebook/docusaurus#5246] [Q&A] [answered]
I updated docusaurus to beta.4. Suddenly I started getting an issue:
```
A validation error occurred.
The validation system was added recently to Docusaurus as an attempt to avoid user configuration errors.
We may have made some mistakes.
If you think your configuration is valid and should keep working, please open a bug report.

ValidationError: The themeConfig.sidebarCollapsible has been moved to docs plugin options. See: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs
```

Then I followed the same instructions and added the plugin `@docusaurus/plugin-content-docs`. I added the configuration in my docusaurus.config.js as:

```js
[
      '@docusaurus/plugin-content-docs',
      {
        sidebarCollapsible: false,
      },
    ]
```
inside the plugin section.

And now I am getting:

> Error: Plugin "docusaurus-plugin-content-docs" is used 2 times with id default.
To use the same plugin multiple times on a Docusaurus site, you need to assign a unique id to each plugin instance.

Please help.

**Comment by khushal87** [comment, marked as answer]
@slorber @Josh-Cena  looking for your help.

**Reply by slorber** [reply]
@khushal87 if you use the preset this config must be defined at the preset level:

```jsx
{ 
      docs: {
        ...
        sidebarCollapsible: false,
        ...
      },
}
```

**Reply by Josh-Cena** [reply]
Do we need to improve the docs on this? Or do we put enough trust in the users

**Reply by khushal87** [reply]
So, there should be no plugin import, right? Only this should be defined under docs under preset? The docs seem to be different.

**Reply by Josh-Cena** [reply]
> So, there should be no plugin import, right?

When you use `preset-classic`, it already comes with a `plugin-docs`. All your configurations under `docs: {}` would be `plugin-docs` options

**Reply by slorber** [reply]
@khushal87 the docs config can be used on both the docs plugin and the classic preset, it's the exact same config, we can't really document it twice in the same way because it would be the exact same doc anyway

**Reply by khushal87** [reply]
Okay, thanks for the update and reply @slorber @Josh-Cena . That was helpful. :)

**Reply by slorber** [reply]
We document it here: https://docusaurus.io/docs/presets#docusauruspreset-classic

I agree this can be confusing, we should probably move the preset doc to the API section, and maybe highlight in each plugin that if they use the preset they can also declare this config through the preset they already use

**Reply by khushal87** [reply]
Exactly @slorber. This was the point of confusion for me.
